daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dsom/DFDLStatementMixin.scala

```diff
@@ -49,12 +49,13 @@ trait DFDLStatementMixin extends ThrowsSDE { self: AnnotatedSchemaComponent =>
 
```

Comment:
Consider split and rename: ProvidesDFDLStatementsMixin, and ResolvesDFDLStatementsMixin. Terms would mix in the latter, local and global defining forms would mix in the former.